remoteworkitem/trackeritem_repository.go

```diff
@@ -62,7 +62,7 @@ func lookupIdentities(ctx context.Context, db *gorm.DB, remoteWorkItem RemoteWor
 	identityRepository := account.NewIdentityRepository(db)
 	//spaceSelfURL := rest.AbsoluteURL(goa.ContextRequest(ctx), app.SpaceHref(spaceID.String()))
 	workItem := workitem.WorkItem{
-		ID:      remoteWorkItem.ID,
+		// ID:      remoteWorkItem.ID,
```

Comment:
I think the remote work item's ID (e.g. github issue ID) can become the new number, no?